src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -913,9 +877,13 @@ function _accumulateInBuffer(
   // A number between 0 and 1 for sample ratio. It changes depending on
   // the CPU usage if it's given. If not, it uses 1 directly.
   const beforeSampleCpuRatio =
-    cpuBeforeSample === null ? 1 : cpuBeforeSample / maxThreadCPUDelta;
+    cpuBeforeSample === null
+      ? 1
+      : cpuBeforeSample / ensureExists(maxThreadCPUDelta);
```

Comment:
Are these `ensureExists` calls needed?

Reply:
Ah, I was experimenting with nullable max values and I forgot to remove them afterwards. They are not needed indeed. Removed them.